gradle-baseline-java-config/resources/checkstyle/checkstyle.xml

```diff
@@ -298,6 +298,10 @@
             <property name="format" value="static enum"/>
             <property name="message" value="Redundant ''static'' modifier."/>
         </module>
+        <module name="RegexpSinglelineJava">
+            <property name="format" value="(\/\/TODO)|(\/\/ TODO:)|(\/\/ TODO\(\))|(\/\/ TODO\(.*\):[^ ])|(\/\/ TODO\(.*\)[^:])"/>
```

Comment:
played with a bunch of examples in http://www.regexplanet.com/advanced/java/index.html - seems good to me